pkg/dto/dto.go

```diff
@@ -9,6 +9,17 @@ import (
 	"strings"
 )
 
+var (
+	// UserAgentOut for outgoing requests (without libraries). The Git Hash will be replaced by main.go.
+	UserAgentOut      = "Poseidon/" + UserAgentVCSPlaceholder + " Go-http-client/1.1"
```

Comment:
Currently, we only have the Watchdog doing requests without a library. I'm unsure if it's worth the complexity of teaching the libraries how to use our User Agents while we currently have no benefit from doing so 🤷

Reply:
Alright, fine for me. I assume that all Nomad requests are handled by their library anyway, so that's fine :+1:.